packages/react-i18n/package.json

```diff
@@ -34,6 +34,8 @@
     "intl-pluralrules": "^0.2.1"
   },
   "dependencies": {
+    "@babel/template": "^7.6.0",
+    "@babel/traverse": "^7.6.2",
```

Comment:
Hmm, I'm not sure if these are strictly dependencies. Aren't they just used for types? Babel should generally be a peer/ optional dependency IMO